sdk/python/setup.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup
+
+NAME = 'kfmd'
+VERSION = '0.0.1'
+
+REQUIRES = [
+]
+
+setup(
+    name=NAME,
+    version=VERSION,
+    description='KubeFlow Metadata SDK',
+    author='google',
+    install_requires=REQUIRES,
+    packages=[
+        'kfmd',
+        'kfmd.swagger_client',
```

Comment:
Should we check in the swagger client? (if so, we can do it later)

If we don't check in the generated code in the source repo, should we generate it before publishing the package?

Reply:
The current build.sh requires the swagger client to be generated before the kfmd package can be built. In this case, the generated swagger client is not checked in the source repo but the code will be in the kfmd package.

Reply:
Sounds good. I'm leaning towards checking in the generated code under source control but we can do it later.